docs/stdlib-os.md

```diff
@@ -131,6 +133,14 @@ cmd := exec.command("echo", ["foo", "bar"])
 output := cmd.output()
 ```
 
+## FileInfo
+
+- `name`: name of the file the info describes
+- `mtime`: time the file was last modified
+- `size`: file size in bytes
+- `mode`: file permissions as in int, comparable to octal permissions
+- `directory`: boolean indicating if the file is a directory
```

Comment:
wasnt sure how you want these as the other examples only show methods so just tried this to see